core/unit_test/TestExecutionEnvironmentNonInitializedOrFinalized.cpp

```diff
@@ -140,4 +140,69 @@ TEST_F(ExecutionEnvironmentNonInitializedOrFinalized_DeathTest, views) {
       matcher2);
 }
 
+#define DEATH_TEST_EXPECTED_STRING \
+  "Invocation of Kokkos function with uninitialized execution space"
```

Comment:
1. This is quite generic, please try to say which parallel construct was invoked.
2. Don't use a macro here.